operations/README.md

```diff
@@ -11,6 +11,15 @@ This section describes all operations supported by the API, organised here by th
 | [Add addresses](addresses.md#add-addresses) | Adds a new address to the system and assigns it to a specified account |
 | [Update addresses](addresses.md#update-addresses) | Updates an existing address in the system assigned to a specified account |
 
+## Account notes
+
+| <div style="width:200px">Operation</div> | Description |
+| :-- | :-- |
+| [Get all account notes](accountnotes.md#get-all-account-notes) | Returns all the account notes associated with the specified accounts within the chain. |
+| [Add account notes](accountnotes.md#add-account-notes) | Adds new account notes to the system and assigns it to specified accounts. |
```

Comment:
```suggestion
| [Add account notes](accountnotes.md#add-account-notes) | **Restricted!** Adds new account notes to the system and assigns it to specified accounts. |
```
I've suggested adding this, because the description says that the operation is subject to change. We normally mark such operations as `restricted`; when the operation becomes stable then the restriction can be removed.